app/Models/User.php

```diff
@@ -802,7 +802,7 @@ public function getHomeDepartmentAttribute(): ?string
     {
         $uid = $this->uid;
 
-        return Cache::remember('home_department_'.$uid, now(), static function () use ($uid): ?string {
+        return Cache::remember('home_department_'.$uid, now()->addDay(), static function () use ($uid): ?string {
```

Comment:
It seems like the second parameter of this function, `now()...`, is the duration to store the item in the cache if the item isn't present.  Unless you can give it a date, but that's not what it looks like based on how [the docs](https://laravel.com/docs/8.x/cache#retrieve-store) describe this function.  How I'm reading it, it seems like you'd want `24*60*60` for the seconds value.

Reply:
The [source](https://github.com/laravel/framework/blob/9.x/src/Illuminate/Support/Facades/Cache.php#L22) indicates that a `\DateTimeInterface` can be provided in that field.